proj-android/PowerAuthLibrary/build.gradle

```diff
@@ -75,5 +79,21 @@ publishing {
     }
 }
 
-
-
+bintray {
+    user = System.getenv('BINTRAY_USER')
+    key  = System.getenv('BINTRAY_API_KEY')
+    publications = ['maven']
+	pkg {
+		repo = 'PowerAuth'
+		name = 'powerauth-android-sdk'
```

Comment:
Thanks for hint. I'll use a separate properties for these values. In fact, that "name" has slightly different meaning than `ARTIFACT_ID`. It's just a package name where the archive will be uploaded.